openshift/aws-resource-exporter.yaml

```diff
@@ -88,6 +88,24 @@ objects:
       port: 9115
       targetPort: 9115
     type: ClusterIP
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
```

Comment:
please add a `qontract.recycle: "true"` annotation, to force a restart of the pods when this configmap changes